pkg/registry/clusterrolebinding/policybased/storage.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
> Yeah I saw those.
> 
> Was thinking that if you wanted to create a storage with checking you'd use the policybased storage, and for bootstrapping we'd use the storage in the etcd package, since both can be used to create a {{ objecttype }}.Registry

We were worried about overlapping interfaces allowing someone to accidentally use the wrong method somewhere.  Non-overlapping interface for making changes without escalation prevents that from happening.